<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Classes</class>
 <widget class="QWidget" name="Tab_Classes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="tree_classes">
      <property name="rootIsDecorated">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbl_className">
           <property name="text">
            <string>Class:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_className">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_classId">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_classId">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_parentId">
           <property name="text">
            <string>Parent:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_classId_parent">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_levelFilter">
           <property name="text">
            <string>Levels: [</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_minLvl">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_points">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>..</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_maxLvl">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_closeBr">
           <property name="text">
            <string>]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_hidePassive">
           <property name="text">
            <string>Hide passive</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_skills">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
